### ISSUE TITLE
Events does not trigger when i ko the templates.

### ISSUE THREAD

**Issue by GagandeepSidhu** [formvalidation/support#413]
Hi, I am trying to validate the form fields which i get get dynamically using ko on drop down select and i can see all the html elements for your plugin but for some reason it does not trigger the error or success events. I tried using dynamic fileds validation on change validation and tried revalidate but nothing happens. Any idea how should i proceed. Any help would be appreciated. Thanks

 <form id="firewallRuleForm" class="form-horizontal " data-bind="with: formModel">

        <div class="form-group">
            <label class="col-xs-2 control-label">Rule type</label>
            <div class="col-xs-4">
               <select name="selectFirewallRule" class="form-control select2-select" style="width:208px" data-bind="value: selectedFirewallRuleType, valueAllowUnset: true, select2: {}">
                 <option value="">Choose Rule type</option>
                 <option value="Standard">Standard</option>
                 <option value="Rate limit">Rate limit</option>
                 <option value="Packet Rewrite">Packet Rewrite</option>
                 <option value="Rate limit queue Bypass">Rate limit queue Bypass</option>
                 <option value="Wifi">Wifi</option>
               </select>
            </div>
        </div>

      <div id="standardRuleForm">

        <div class="form-group" >
           <label class="col-xs-2 control-label">Rule name</label>
           <div class="col-xs-4 ">
              <input type="text" class="form-control" name="firewallRuleName" placeholder="Enter the name of the rule" data-bind="textInput: ruleName"/>
           </div>
           
           <div class="col-xs-offset-2 col-xs-2">
                <div class="checkbox">
                  <label><input type="checkbox">Log Matches? </label>
                </div>
           </div>
           
        </div>
        
        <div class="form-group">
                            <label class="col-xs-2 control-label">Set</label>
                            <div class="col-xs-4 selectContainer">
                            <select class="form-control select2-select" name="setPacketWrite" data-bind="value : packetRewriteSet , valueAllowUnset: true, select2: {}">
                                  <option value="">Choose packet</option>
                                  <option value="MSS">MSS</option>
                                  <option value="WSCALE">WSCALE</option>

                              </select>
                            </div>
                        </div>
        

  $('#firewallRuleForm')
            .find('[name="selectFirewallRule"]')
            .change(function(e){
            console.log("getting this");
            $('#firewallRuleForm').formValidation({
                          framework: 'bootstrap',

                          icon:{
                            valid: 'glyphicon glyphicon-ok',
                            invalid: 'glyphicon glyphicon-remove',
                            validating: 'glyphicon glyphicon-refresh'
                          },
                          fields: {
                              selectFirewallRule: {
                                  validators: {
                                      notEmpty: {
                                          message: 'The Alias is required'
                                      }

                                  }
                              },
                              firewallRuleName:{

                                validators: {
                                    notEmpty: {
                                        message: 'The Alias is required'
                                    }

                                }
                            },
                           setPacketWrite:{

                             validators: {
                                 notEmpty: {
                                     message: 'The Alias is required'
                                 }

                             }
                         }

                          }
                        });

**Comment by nghuuphuoc** [comment]
I haven't had chances to give ko a try but you should move `$('#firewallRuleForm').formValidation(...)` to out of the `change` event handler.
It should be

```javascript
$('#firewallRuleForm')
    .formValidation({
        fields: ....
    })
    .on('change', '[name="selectFirewallRule"]', function() {  // <--- Handle the change event
        console.log("Do you see this line?");
        ...
    });
```

**Comment by GagandeepSidhu** [comment]
.on('change', '[name="selectFirewallRule"]' , function(e, data){
        console.log("yo man");
        $("#firewallRuleForm").formValidation('revalidateField', 'firewallRuleName');
I am trying to revalidate but nothing happens and i tried this ab the event triggers but still no validation and after moving it to the bottom now it dint even instantiate formvalidation for the fields after <!-- ko if: (selectedFirewallRuleType() ---

   .on('keyup', '[name="password"]', function() {
            var isEmpty = $(this).val() == '';
            $('#enableForm')
                    .formValidation('enableFieldValidators', 'password', !isEmpty)
                    .formValidation('enableFieldValidators', 'confirm_password', !isEmpty);

            // Revalidate the field when user start typing in the password field
            if ($(this).val().length == 1) {
                $('#enableForm').formValidation('validateField', 'password')
                                .formValidation('validateField', 'confirm_password');
            }
        });
can i trigger them myself(maually)  to make sure the event success or error triggers on some condition ? I can write the contions my self i just want it show that fancy stuff on success and error

**Comment by nghuuphuoc** [comment]
> can i trigger them myself(maually) to make sure the event success or error triggers on some condition

You should trigger the `success.field.fv` or `err.field.fv` events.
See http://formvalidation.io/settings/#event-field

The list of examples at the bottom of section above might be useful.
